src/brer/pair_data.py

```diff
@@ -32,15 +38,51 @@ class PairData:
 
     Essential pair data to support BRER MD plugin code. All data must be
     provided when the object is initialized. (Fields are read-only.)
+
+    Fields here correspond to the fields for each named pair
+    (JSON *objects*) in a :file:`pair_data.json` file (the *pairs_json* argument
+    of :py:func:`~brer.run_config.RunConfig()`).
     """
     # Historically, *name* has sometimes been provided as a positional argument,
     # but this led to ambiguity in the source of the final value of the field.
     # We define the *name* field first in case we overlook some legacy usage.
     name: str = dataclasses.field(**field_kwargs)
+    """Identifier for the pair of sites on the molecule.
+    
+    Conventionally, this is a string constructed from the two integer indices
+    of relevant atoms in the molecular model that best correspond to the
+    labeled sites of interest.
```

Comment:
This description did not seem to be true. How does this look?
```suggestion
    This string is chosen by the researcher. For example, the name may
    include identifiers for the two residues in a scheme that can be easily
    cross-referenced with experimental data.
```